src/services/openletters/index.jsx

```
@@ -471,6 +473,470 @@ export const AddOpenLetter = ({
   );
 };
 
+export const EditOpenLetter = ({
+  setEditModal,
```

ESLint

[error] 'setEditModal' is missing in props validation

(react/prop-types)